doc/release/1.13.0-notes.rst

```diff
@@ -189,6 +189,13 @@ if all fields that are non-masked in both are equal, thus making the behaviour
 identical to what one gets by comparing an unstructured masked array and then
 doing ``.all()`` over some axis.
 
+np.matrix with booleans elements can now be created using the string syntax
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+``np.matrix`` failed whenever one attempts to use it with booleans, e.g.,
+``np.matrix('True')``. Now, this works as expected. Additionally, a
+``ValueError``, rather than a ``TypeError``, is now throw whenever an invalid
```

Comment:
This isn't really true. `np.matrix('(')` fails with a `SyntaxError`, and `np.matrix('{{}}')` fails with a `TypeError`.

The previous errors were also pretty random